intranet/templates/docs/accounts.html

```diff
@@ -18,27 +18,30 @@ <h3 class="block-center-heading" id="first-week-checklist">First Week Checklist<
       <li>Reset your CSL password by following the steps in the <a href="#setting-passwords">"Setting/Resetting Passwords" section</a>. You will obtain your TJ username during this process.</li>
       <li>Verify you can log in to Ion.</li>
       <li>Ensure that you set up mail forwarding for your @tjhsst.edu email address during the Ion welcome process to an email you regularly check.</li>
+      <!--
+        NOTE: Temporarily removed for virtual instruction.
       <li>Visit <a href="https://ion.tjhsst.edu/preferences">Ion preferences</a> to set your bus route if applicable.</li>
-      <li>Verify that you can log in to <a href="https://webmail.tjhsst.edu">TJHSST Webmail</a> to check your TJ email address.</li>
-      <li>Sign up for some eighth period activities on Ion.</a>
+      -->
+      <li>Verify that you can log in to <a href="https://webmail.tjhsst.edu">TJHSST Webmail</a> in order to check your TJ email address.</li>
+      <li>Sign up for eighth period activities on Ion.</a>
     </ul>
     <br>
 
     <h3 class="block-center-heading" id="setting-passwords">Setting/Resetting Passwords</h3>
-    <p>You can reset your password by visiting <a href="https://resetter.tjhsst.edu">https://resetter.tjhsst.edu</a>.</p>
-    <ul>
+    <p>You can reset your CSL password by visiting <a href="https://resetter.tjhsst.edu">https://resetter.tjhsst.edu</a>.</p>
+    <ol>
       <li>Visit <a href="https://resetter.tjhsst.edu">https://resetter.tjhsst.edu</a> on a school or personal device.</li>
-      <li>Click the "Authenticate" button.</li>
+      <li>Click the "Authenticate" button. If you are already authenticated, you may be immediately proceed to step 4.</li>
```

Comment:
```suggestion
      <li>Click the "Authenticate" button. If you are already authenticated to your FCPS Google account, you may be skipped to step 4.</li>
```

"you may be immediately proceed" is not grammatically correct.